NOTICE.txt

```diff
@@ -20689,27 +20689,27 @@ Licence type (autodetected): MIT
 
 Contents of probable licence file $GOMODCACHE/github.com/microsoft/wmi@v0.25.1/LICENSE:
 
-    MIT License

```

Comment:
After running make notice, I still see uncommitted changes. Could you please run make notice again and commit the changes? Thanks!

Reply:
Thanks, NOTICE looks good now!